src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -52,6 +52,9 @@ class OpenEdxVars(BaseModel):
     lms_domain: str
     logo_url: str
     marketing_site_domain: str
+    mit_learn_base_url: str
+    mit_learn_api_base_url: str
+    mit_learn_logo: str
```

Comment:
These should be `str | None = None` to make them optional, otherwise it will error if no value gets passed to the model constructor.

Reply:
updated in https://github.com/mitodl/ol-infrastructure/pull/3179/commits/5af2f01451daa80b1718c0e27b3f885ffb6b4658 as per old code `Optional[str] = None`